auth/README.md

```diff
@@ -1,38 +1,39 @@
 # Auth Package
 
-Auth package is used for request authentication and authorization for user to service
-and service to service communication. Auth is used for scenarios that require stricter
-validations, and usage restrictions. `auth.Client` provided by the package provides seamless
+Use `package auth` for request authentication and authorization for user-to-service
+and service-to-service communication. Use auth for scenarios that require stricter
+validations and usage restrictions. `auth.Client`, included in the package provides seamless
 integration with presently written modules in the service framework. As a middleware, Auth
 provides additional restrictions on who can access a service, and who can be
 authenticated to access the service.
-The Auth package doesn't dictate how authentication and authorization should work, or which
-algorithm the security service should use, but only allows client integration with the service framework.
+The Auth package doesn't dictate how authentication and authorization should work, nor which
```

Comment:
Auth --> auth
(Unless you think there's a strong reason to capitalize packages. I'm still learning Go's style conventions, but I think lowercase is OK.)